sdcm/sct_events/events_device.py

```diff
@@ -60,7 +61,7 @@ def __init__(self, _registry: EventsProcessesRegistry):
         self._sub_port = multiprocessing.Value(ctypes.c_uint16, 0)
         self._queue = multiprocessing.Queue()
         self._raw_events_lock = multiprocessing.RLock()
-
+        self._is_alive_flag = multiprocessing.Semaphore()
```

Comment:
Why do we need another attribute which says us that a device is running? Is it not enough to check `self._running`?

Reply:
You are right I have overlooked it.